index.html

```diff
@@ -44,5 +45,126 @@ <h3>LET’S CONNECT</h3>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
+    
+
+    <div class="card">
+      <img src="assets/images/tonic2.svg" alt="tonic2">
+      <div class="tonic">
+        <h3>Tonic</h3>
+        <ul>
+          <li><a class="Canopy" href="#">Canopy <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
+
+
+    <div class="card">
+      <img src="assets/images/poststories2.svg" alt="tonic1">
+      <div class="tonic">
+        <h3>Multi-Post <br>
+          Stories</h3>
+        <ul>
+          <li><a class="Canopy" href="#">Canopy <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> Back End Dev <img src="assets/images/Counter.svg" alt=""></a></li>
+          <li><a href="#"> 2015</a></li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="lang">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">Javascript</a></li>
+        </ul>
+        <button>See project</button>
+    </div>
+  </section>
+
+  <!--About me section-->
+  <section class="about-me">
+    <div class="top-about-me">
+      <h1>About Myself</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="let-connect">
+        <h3>LET’S CONNECT</h3>
+        <ul class="social_link">
+          <li><a href="#"><i class="fa fa-twitter fa-lg"></i></a></li>
+          <li><a href="#"><i class="fa fa-linkedin fa-lg"></i></a></li>
+          <li><a href="#"><i class="fa fa-medium fa-lg"></i></a></li>
+          <li><a href="#"><i class="fa fa-github-alt fa-lg"></i></a></li>
+          <li><a href="#"><i class="fa fa-angellist fa-lg"></i></a></li>
+        </ul>
+        <button>Get my resume</button>
+        <div>
+          <ul class="skills">
+            <li> <p class="list"><span>Language</span>  <i class="fa-solid fa-chevron-down"></i></p> 
```

Comment:
[OPTIONAL]
- [ ] Please can you kindly add the arrows which are currently missing in the design of your portfolio. it is impoertant to show that your pay attention to detials as a frontend developer when building projects that has a specific design so please i kindly suggest that you add these arrows as shown in the figma template below.
- [ ] Great job from you implementing all of the previous requirements by the earlier code reviewers :+1: :grin: 
- figma design
![arr-f](https://user-images.githubusercontent.com/89055502/236588617-2283f50b-839f-40ef-832a-d506fa4b1678.png)
- your design
![arr-s](https://user-images.githubusercontent.com/89055502/236588633-80cef0c2-2b70-4c5e-88d2-57d28011616b.png)